uint16 state_agent1
uint16 state_agent2
uint16 state_agent3

uint16 mode_agent1
uint16 mode_agent2
uint16 mode_agent3

int16 targetx_agent1
int16 targety_agent1
int16 targett_agent1

int16 targetx_agent2
int16 targety_agent2
int16 targett_agent2

int16 targetx_agent3
int16 targety_agent3
int16 targett_agent3

int16 localization_posx
int16 localization_posy
int16 localization_post

int8 flaglocalization

int16 global_ballposx
int16 global_ballposy
int8 ball_detect


